pandas/io/parsers.py

```diff
@@ -102,11 +102,14 @@
     List of column names to use. If file contains no header row, then you
     should explicitly pass ``header=None``. Duplicates in this list will cause
     a ``UserWarning`` to be issued.
-index_col : int, sequence or bool, optional
-    Column to use as the row labels of the DataFrame. If a sequence is given, a
-    MultiIndex is used. If you have a malformed file with delimiters at the end
-    of each line, you might consider ``index_col=False`` to force pandas to
-    not use the first column as the index (row names).
+index_col : int, str, sequence of int / str, or False, default ``None``
+  Column(s) to use as the row labels of the ``DataFrame``, either given as
+  string name or column index. If a sequence of int / str is given, a
+  MultiIndex is used. Columns used for the index (row names) are dropped from
+  the actual columns of the input dataframe. They are accessible via
+  ``.index``. (Note: ``index_col=False`` can be used to force pandas to *not*
```

Comment:
Can you format this similar to the user_guide version?

Reply:
Like this? (I'm not sure why this is duplicated across code comments and documentation in the first place instead of being included from the code when compiling the documentation, but if it's to be fixed, it should probably be in another pull request.)